src/main/java/repo/impl/CouchUserRepo.java

```diff
@@ -33,4 +33,8 @@ public CouchUser getUserByUsername(String username) {
         }
         return users.get(0);
     }
+
+    public String couchIsUp() {
+        // Test whether requests to couch are going
```

Comment:
I am trying to query `couchDbClient` for `couch_host/couch_db_name` and test whether the response JSON contains the key-val `{'db_name': <couch_db_name>}`.

Can you give an example of how to query `couchDbClient` by providing a URL? And an example of looking up properties of JSON objects in Java.